client/index.html

```diff
@@ -468,6 +474,64 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
+						<h2>Autocompletion</h2>
+
+						<p>
+							The Lounge offers automatic completion of nicknames, channels, <kbd>/</kbd>-commands, emoji, and text colors.
```

Comment:
I don't think `<kdb>-</kbd> commands` is necessary; just `commands` is sufficient